components/card/arrow/src/index.js

```diff
@@ -14,12 +14,16 @@ const CardArrow = ({media, text, icon, linkFactory: Link, link}) => {
   return (
     <div className="sui-CardArrow">
       <Link className="sui-CardArrow-link" href={link}>
-        <div className="sui-CardArrow-img">
-          <AtomImage src={media.src} alt={media.alt} />
-        </div>
+        {media && (
+          <div className="sui-CardArrow-img">
+            <AtomImage src={media.src} alt={media.alt} />
```

Comment:
I know its not a PR related thing, but what about adding the title attribute?

Reply:
well, in other cards maybe makes more sense, here I doubt that a tiny image has a SEO value.

Reply:
well, it has not a role='presentation' or alt=''. This means this image is considered important for the content. Because of this, a title will help @ivanmlaborda